src/base/memory_monitor_common.h

```diff
@@ -26,54 +26,59 @@
 
 #include <cstdint>
 
-typedef struct memmon_mem_stat
+typedef enum mmon_module_id
+{
+  MMON_MODULE_LAST = 1
```

Comment:
example
```suggestion
  //MMON_MODULE_HEAP = 1,
  MMON_MODULE_LAST
```

It seems that there is no need to assign a value to MMON_MODULE_LAST,
because MMON_MODULE_LAST is likely to be used to see the total number of modules, it would be cumbersome to have to update this value every time a module is added

Reply:
I removed it because I thought adding an enum for the heap module was out of the scope of this story. If you think adding comments is okay, I will add them.

Reply:
the comment was simply an example provided to aid in understanding. 
The intention was to clarify that there is no need to assign a value to MMON_MODULE_LAST :)

Reply:
Done.